src/ck-core/ckfutures.h

```diff
@@ -84,7 +84,7 @@ namespace ck {
       CkSendToFuture(handle_, msg);
     }
 
-    bool probe() { return CkProbeFuture(handle_); }
+    bool is_ready() { return CkProbeFuture(handle_); }
```

Comment:
In reference to Prof. Kale's comment on the proposed change, can we keep both `is_ready` and `probe` as synonyms to keep it similar to both C++ future's member function `is_ready`, and MPI's `probe` function?

Reply:
Probe is retained within the C-like interface, which can serve as a frame-of-reference for MPI users. For this reason, and the fact that they are documented side-by-side, I do not think an alias is necessary within the C++ interface.

Reply:
Thanks for the clarification. LGTM thanks!